test/components/connectors/test_openapi_service.py

```diff
@@ -175,3 +175,65 @@ def test_run_with_mix_params_request_body(self, openapi_mock, test_files_path):
         result = connector.run(messages=messages, service_openapi_spec=spec)
         response = json.loads(result["service_response"][0].content)
         assert response == "Hello, John"
+
+    @patch("haystack.components.connectors.openapi_service.OpenAPI")
+    def test_run_with_complex_types(self, openapi_mock, test_files_path):
```

Comment:
The PR description lists as changes

> "In openapi_service.py, the connector now supports cases where request parameters or body properties might be missing in the invocation arguments."

I expected to find a test here `test_run_with_request_params_missing_in_invocation_args` or `test_run_with_body_properties_missing_in_invocation_args`

Reply:
Yep will add that test.